### ISSUE TITLE
Code spans in table heading not supported

### ISSUE THREAD

**Issue by richanna** [cabo/kramdown-rfc2629#66]
Surrounding a table heading in back ticks results in a non-fatal error message, and an empty table heading.

e.g.,
The table heading:
`| ``foo`` | bar`

Produces the error:
`*** lost markup <spanx style="verb">foo</spanx> in table heading`

And renders equivalently to:
`| | bar`

**Comment by cabo** [comment]
Right, that was a limitation of RFCXMLv2.  We can fix that by outputting tables in RFCXMLv3.
